libsnark/zk_proof_systems/plonk/prover.tcc

```diff
@@ -527,40 +527,40 @@ round_four_out_t<ppT>::round_four_out_t(
 {
 }
 
-/// Prover Round 4
-///
-/// INPUT
-/// \param[in] W_polys_blinded: blinded witness polynomials (from
-///            Round 1)
-/// \param[in] z_poly_xomega: the polynomial z(x*w) i.e. z(x) shifted
-///            by w (from Round 3)
-/// \param[in] t_poly_long: the quotient polynomial t(x) (see Round 3,
-///            pp28 [GWC19]) (from Round 3)
-/// \param[in] srs: structured reference string containing also
-///            circuit-specific information
-///
-/// OUTPUT
-/// \param[out] a_zeta, b_zeta, c_zeta: the blinded witness
-///             polynomials a(x), b(x), c(x) (denoted by
-///             W_polys_blinded[] output from Round 1) evaluated at
-///             x=zeta i.e. a(z), b(z), c(z)
-/// \param[out] S_0_zeta, S_1_zeta: the permutation polynomials
-///             S_sigma_1(x), S_sigma_2(x) from the common
-///             preprocessed input (see [GWC19], Sect. 8.1) evaluated
-///             at x=zeta i.e. S_sigma_1(z), S_sigma_2(z)
-/// \param[out] z_poly_xomega_zeta: the polynomial z(x*w) i.e. z(x)
-///             shifted by w (output from Round 3) evaluated at x=zeta
-///             i.e. z(zeta*w)
-/// \param[out] t_zeta: the quotient polynomial t(x) output from Round
-///             3, see pp28 [GWC19]) evaluated at x=zeta
-///             i.e. t(z). IMPORTANT! the original Plonk proposal
-///             [GWC19] does not output this parameter t_zeta. The
-///             Python reference implementation \[PlonkPy] does, so we
-///             do the same in order to match the test vectors. TODO
-///             can remove t_zeta in the future
-/// \param[out] transcript_hasher: accumulates the communication
-///             transcript into a buffer to be hashed after prover
-///             rounds 1,2,3,4,5 (cf. fiat-shamir heuristic).
+// Prover Round 4
```

Comment:
No problem having these as doc comments, but in that case, let's put them consistently in the header only.